src/replica_server/replication_lib/replica_init.cpp

```diff
@@ -269,9 +269,8 @@ error_code replica::init_app_and_prepare_list(bool create_new)
                         _private_log->max_commit_on_disk(),
                         finish_time - start_time
                         );
+
                     _private_log->check_valid_start_offset(get_gpid(), _app->init_info().init_offset_in_private_log);
-                    set_inactive_state_transient(true);
```

Comment:
but this stmt also appears in replica_stub.cpp:361, which will overwrite all old values, so it's unnecessary to set here.